src/controllers/erc721/metadata/images/post.controller.ts

```diff
@@ -66,7 +69,6 @@ const controller = async (req: Request, res: Response) => {
             logger.info(`INVALID FILE TYPE, FILE SKIPPED: ${file}`);
             continue;
         }
-
         const promise = new Promise(async (resolve, reject) => {
```

Comment:
@valeriagrazzini This is an incorrect pattern, you don't have to "promisify" a promise (the async executor thats already there). I've added a comment with some docs about this a couple of days ago.